lib/sirsi_holding.rb

```diff
@@ -149,7 +149,7 @@ def to_item_display_hash
   private
 
   # Call number normalization ported from solrmarc code
-  def normalize_call_number(call_number = '')
+  def normalize_call_number(call_number)
```

Comment:
Couldn't this `call_number` be nil too? Wouldn't lines 155-158 need some handling?

Reply:
Isn't it possible to have a call_number_type set, but end up with nil call number? I don't see `call_number_type` checking anything besides the provided schema.

Reply:
Maybe, but we've never seen data like that. I doubt we ever will. When the call_number has been nil (situation we're fixing), the type is also nil.